<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Debug_Laser_Section">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Debugging Laser</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>291</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Debug_Laser_On">
       <property name="text">
        <string>Laser Left On</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Debug_Laser_Off">
       <property name="text">
        <string>Laser Left Off</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>671</width>
      <height>252</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="Original_img">
       <property name="minimumSize">
        <size>
         <width>330</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">Border: 10px solid black</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Thresh_img">
       <property name="minimumSize">
        <size>
         <width>330</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 10px solid black</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Cancel_Scan_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>280</y>
      <width>217</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="save_label">
       <property name="text">
        <string>Set Mesh Destination:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Scan_Save_Button">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>281</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="progressBar_label">
       <property name="text">
        <string>Scanning Progress</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Debug_Motor_Section">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>210</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Debugging Motor</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>230</y>
      <width>291</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="Debug_Motor_Left">
       <property name="text">
        <string>Motor Left</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Debug_Motor_Right">
       <property name="text">
        <string>Motor Right</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Error_label">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Connected!</string>
    </property>
   </widget>
   <widget class="QLabel" name="No_Error_label">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connected!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="retry_Connection_button">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>114</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Retry</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>221</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lPort_Connection_label">
       <property name="text">
        <string>Port Connection:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Connection_Colour_Label">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>291</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="Debug_laser_Right_on">
       <property name="text">
        <string>Laser Right On</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Debug_Laser_Right_off">
       <property name="text">
        <string>Laser Right Off</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>Debug_Laser_Section</zorder>
   <zorder>Cancel_Scan_button</zorder>
   <zorder>Debug_Motor_Section</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>Error_label</zorder>
   <zorder>No_Error_label</zorder>
   <zorder>retry_Connection_button</zorder>
   <zorder>Debug_laser_Right_on</zorder>
   <zorder>Debug_Laser_Right_off</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1039</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
